src/btree/bt_vrfy_dsk.c

```diff
@@ -346,12 +346,14 @@ static int
 __verify_row_key_order_check(
   WT_ITEM *last, uint32_t last_cell_num, WT_ITEM *current, uint32_t cell_num, WT_VERIFY_INFO *vi)
 {
+    WT_BTREE *btree;
     WT_DECL_ITEM(tmp1);
     WT_DECL_ITEM(tmp2);
     WT_DECL_RET;
     int cmp;
 
-    WT_RET(__wt_compare(vi->session, S2BT(vi->session)->collator, last, current, &cmp));
+    btree = S2BT(vi->session);
```

Comment:
Is `S2BT` always ok or should we use the safe version `S2BT_SAFE`?

Reply:
S2BT should be okay because the caller of the function uses S2BT.